clusters/app.ci/hive/README.md

```diff
@@ -42,5 +42,270 @@ hiveadmission-9f7df866b-lbcmp       1/1     Running   0          2m16s
 hiveadmission-9f7df866b-zb98l       1/1     Running   0          2m16s
 ```
 
+## Configure DNS for Hive
+### AWS Route53
+base domain: `hive.aws.ci.openshift.org`. We only need set up this once for all clusters on AWS with `hive.aws.ci.openshift.org` as base domain.
+
+Account: https://openshift-ci.signin.aws.amazon.com/console
+
+This account has been used to create ephemeral clusters via installer directly for CI jobs.
+
+* route53 console: Create a zone `hive.aws.ci.openshift.org`. We can see the list of name servers in the NS record after creation.
+* cloud DNS at GCP (`project=openshift-ci-infra`): Under zone `origin-ci-ocp-public-dns` (which manages domain `ci.openshift.org`), create a NS record for `hive.aws.ci.openshift.org` using the the list of name servers from route53 console above.
+
+```bash
```

Comment:
There's `console` hightlighter that's better for command-output snippets. `bash ` is for actual bash syntax.

Reply:
(through the whole doc)

Reply:
Updated.